packages/karma-puppeteer-launcher/src/index.cjs

```diff
@@ -58,14 +63,7 @@ function puppeteerBrowser(baseBrowserDecorator, config) {
     };
 
     // See https://github.com/puppeteer/puppeteer/blob/v3.0.4/docs/api.md#puppeteerlaunchoptions.
-    browser = await puppeteer.launch({
-      product: puppeteerOptions.product,
-      slowMo: puppeteerOptions.slowMo,
-      dumpio: puppeteerOptions.dumpio,
-      headless: puppeteerOptions.headless,
-      devtools: puppeteerOptions.devtools,
-      defaultViewport: puppeteerOptions.defaultViewport,
-    });
+    browser = await puppeteer.launch(puppeteerOptions);
```

Comment:
Was there any option previously missing here or is this just for readability?

Reply:
Right, but there aren't really any new args to pass through that were previously missed, right?

And the new `waitForInitialPage` arg is redundant AFAICT because `true` is the default there anyway.

Reply:
the list of `args` was getting ignored on launch, we set args in `defaultPuppeteerOptions` and then `puppeteerOptions` throws those together with `config.puppeteer` overrides and then uses that new object to launch the browser.  

i'll remove waitForInitialPage cuz it is redundant (i hadn't wanted to run the tests again just for that 1 line of code to dry up)

Reply:
I think mainly because it was not forwarding the `args` defined in `puppeteerOptions`.